### DISCUSSION TITLE
Error in converting typst tables to latex

### DISCUSSION THREAD

**Discussion by psads-git** [jgm/pandoc#9669] [Q&A] [answered]
When trying to convert the following `typst` table to `latex`, I get an error:

```
#table(
  columns: 4,
  [], [Exam 1], [Exam 2], [Exam 3],
  [John], [], a, [],
  [Mary], [], a, a,
  [Robert], b, a, b,
)
```

Could you please help me? Thanks in advance!

**Comment by jgm** [comment, marked as answer]
Well, what is the error?
And are you just converting this table?  If you try with with typst itself, you'll see that it also gives you an error, because `a` and `b` are undefined.

**Reply by psads-git** [reply]
I apologize, as I had not noticed `a` and `b`. Thanks, @jgm!
